src/components/Beers.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect} from 'react';
+import './Beers.css';
+import Nav from './Nav';
+import { Link } from 'react-router-dom';
+
+function Beers() {
+  const [beersArray, setBeersArray] = useState([]);
+
+  useEffect(() => {
+    fetchBeers();
+  }, []);
```

Comment:
Same comment as above, you can remove the dependency array since you're not listening for changing values.